src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWPApplication.cs

```diff
@@ -634,7 +634,7 @@ private ImageSource PlatedImage(BitmapImage image)
                     var brush = new SolidColorBrush(color);
                     var pen = new Pen(brush, 1);
                     var backgroundArea = new Rect(0, 0, width, height);
-                    var rectangleGeometry = new RectangleGeometry(backgroundArea);
+                    var rectangleGeometry = new RectangleGeometry(backgroundArea, 2, 2);
```

Comment:
@enricogior This needs to be 4 actually, not 2.

Reply:
Do we want this as part of this PR or just merge and do a quick spot fix PR?